src/ts/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "allowJs": true,
     "allowSyntheticDefaultImports": true,
-    "baseUrl": ".",
+    "baseUrl": "./src",
```

Comment:
Ah sorry I changed the `baseUrl` to `"."`, so it's easier to use autocomplete in important statement starting with `import from 'src/...'` and you get a dropdown of of options 😆

Reply:
Ah, I use Sublime and my TS plugins are not that extensive; I've tried to organize code so we are mostly importing from the current directory or somewhere in `common` anyways.

Reply:
Got it! What if we add another shortcut to import directly from "common"? Would that help?

```ts
    ...
    "baseUrl": ".",
    ...
    "paths": {
      "src/*": [
        "./src/*"
      ],
      "common/*": [ // <-- this!
        "./common/*"
      ]
    },
```